documents/src/pages/elements/tree.md

```diff
@@ -455,6 +455,50 @@ tree?.addEventListener('value-changed', (event) => {
 });
 ```
 
+## Manipulating item properties
+
+Item properties of Tree could be read and updated programmatically through its [TreeManager](./custom-components/utils/tree-manager) which is available as `manager` property. Retrieve [TreeNode(s)](./custom-components/utils/tree-node) representing each item by calling `getTreeNode()` or `getTreeNodes()` of `manager`. In addition, they could be used for [custom renderer](./custom-components/utils/tree-node#custom-renderer) too.
```

Comment:
Let's move "Custom Renderer" from "Tree Node" to "Tree" & "Tree Select" + revise it content a little as readers focus on Component first rather than "Tree Node" first.